wiki/Tournaments/CIC/2019/en.md

```diff
@@ -0,0 +1,442 @@
+5. Testplayers, referees, and map selectors may not participate as players in this tournament.
+
+### Group stage rules
+
+1. In the Group stage, all qualified players will be situated into different seeds based on their osu!catch pp. The players from each seed will then be drawn into eight groups at random based on the result of a livestreamed [random.org](http://www.random.org/lists/ "random.org") draw.
+2. All players in each group will compete against each other in a single multiplayer lobby using the Battle Royale system. In this system, all players will each in turn pick out two maps from the mappool to be played with the following turnouts after each map:
+   - 1st place: **+2 point(s)**
+   - 2nd place: **+1 point(s)**
+   - 3rd place: **-1 point(s)**
+   - 4th place: **-2 point(s)**
+3. Players are **not allowed** to ban maps in the Group stage.
+4. There is no Lower Bracket in the Group stage.
+5. Rankings of each group are determined by sorting the results of each individual player's performance in the following priority:
+   - Most points obtained.
+   - Winner of the Tiebreaker.
+   - Player(s) who Win By Default.
+   - Whether there are players who are disqualified from the tournament.
+6. Two players who placed on top of their respective groups' rankings based on the criteria listed above will advance to the Knock-out stage.
+
+### Knock-out stage rules 
+
+1. The eight Group stage winners will be matched to the eight Group stage runners-up at random; that is, instead of following a predetermined match-up format (A1 vs H2, B1 vs G2, and so on), all the match-ups will be based on the result of a livestreamed [random.org](http://www.random.org/lists/ "random.org") draw.
+2. Players will compete against each other using the Double Elimination system.
+3. The Double Elimination System works as following:
+   - Players who lose in the Upper Bracket can still play again on the Lower Bracket.
+   - Players who lose in the Lower Bracket will be eliminated from the tournament.
+   - In the Grand Finals, the winner of the the Upper Bracket only needs to win a single match against their opponent in order to clinch the championship title. The winner of the Lower Bracket, however, needs to win two matches and enforce a Bracket Reset against their opponent in order to clinch the championship title.
+4. Players who can compete in the next round are determined by:
+   - In the Round of 16 and the Quarterfinals, each player needs to win 5 points in order to win a match. (Best-of-9)
+   - In the Semifinals and the Finals (except for the Lower Bracket Final and the Grand Final), each player needs to win 6 points in order to win a match. (Best-of-11)
+   - In the Lower Bracket Final and the Grand Final, each player needs to win 7 points in order to win the match. (Best-of-13)
```

Comment:
```suggestion
   - In the Round of 16 and the Quarterfinals, each player needs to win 5 points in order to win a match. (Best of 9)
   - In the Semifinals and the Finals (except for the Lower Bracket Final and the Grand Final), each player needs to win 6 points in order to win a match. (Best of 11)
   - In the Lower Bracket Final and the Grand Final, each player needs to win 7 points in order to win the match. (Best of 13)
```

Reply:
hmm as far as i know the wording of this rule on most of the official tournaments (incl. the World Cups) are being written this way so i think i'll keep these lines as is if you don't mind :

![image](https://user-images.githubusercontent.com/36564236/131824704-be6dd2a1-4852-48e8-9d88-cc3412314055.png)

Reply:
thanks for checking